.circleci/config.yml

```diff
@@ -27,6 +27,7 @@ set_macos_env: &macos_env
       echo 'export GOPATH=$HOME/project' >> $BASH_ENV
       echo 'export GOBIN=$GOPATH/bin' >> $BASH_ENV
       echo 'export PATH=$PATH:$HOME/go/bin:$GOBIN' >> $BASH_ENV
+      echo 'export GO111MODULE=on'
```

Comment:
Suggestion: If you move the code out of $GOPATH/src (simply do not specify a `working_directory` above), this export is not necessary: https://circleci.com/blog/go-v1.11-modules-and-circleci/

Reply:
Yes, I know that. I think this change could be done on a separate PR though